uobtheatre/bookings/models.py

```diff
@@ -719,18 +728,29 @@ def seat_price(self) -> int:
             seat_group=self.seat_group
         ).price
 
-    def check_in(self):
+    @property
+    def checked_in(self) -> bool:
+        """Boolean property for if a ticket is checked in.
+
+        Returns:
+            bool: Whether the ticket is checked in.
+        """
+        return self.checked_in_at is not None
+
+    def check_in(self, user: User):
```

Comment:
This should raise an exception in the event of a ticket being checked in. We probably already handle this exception in the resolver but can we move this here?